lib/netsuite/records/note.rb

```diff
@@ -0,0 +1,36 @@
+module NetSuite
+  module Records
+    class Note
+      OUTGOING = '_outgoing'
+      INCOMING = '_incoming'
```

Comment:
@iloveitaly Yes, gem user can use these constants. Maybe this is better than use magic strings.